app/templates/components/preprint-form-authors.hbs

```diff
@@ -2,9 +2,9 @@
     <div class="col-xs-12 col-md-5 col-division-right">
         <form {{action 'findContributors' 1 on='submit'}}>
             <div class="input-group author-search-box">
-                <input id="author-search-box" type="text" class="form-control searchQuery" onchange={{action (mut query) value="target.value"}} placeholder={{t "components.preprint-form-authors.search.placeholder"}}>
+                <input id="author-search-box" type="text" disabled={{not isAdmin}} class="form-control searchQuery" onchange={{action (mut query) value="target.value"}} placeholder={{t "components.preprint-form-authors.search.placeholder"}}>
```

Comment:
As read/write contribs can access this form section, several places are disabled when the user is not an admin.